microlith/grafana/provisioning/dashboards/bucket-overview.json

```diff
@@ -927,15 +942,14 @@
   "templating": {
     "list": [
       {
-        "allValue": null,
-        "current": {},
-        "datasource": null,
+        "current": {
```

Comment:
Will have to manually go through all dashboards and remove all these default value changes for each variable / update them all to be
`"current": {}` again, because otherwise Grafana tries to pre-populate (not a huge issue but depending on the time the variables update the user might notice, and can be confusing)

Reply:
grumble - didn't we discuss a lint rule for this at some point?